telegram/update.py

```diff
@@ -59,6 +64,8 @@ def __init__(self,
                  inline_query=None,
                  chosen_inline_result=None,
                  callback_query=None,
+                 shipping_query=None,
+                 pre_checkout_query=None,
```

Comment:
For backward compatibility (if some1 for any reason, decided to use `Update` directly) please add the new parameters to the end (just before **kwargs).